create_deb.sh

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+PROGRAM_NAME=sdlhmi
+VERSION=6.1.0
+PROGRAM_NAME=$PROGRAM_NAME-$VERSION
+
+rm -rf $PROGRAM_NAME
+mkdir $PROGRAM_NAME
+cp -r  deb/* $PROGRAM_NAME/
+mkdir -p $PROGRAM_NAME/usr/bin/
+mkdir -p $PROGRAM_NAME/usr/share/sdlhmi
+mkdir -p $PROGRAM_NAME/usr/share/doc/sdlhmi
+cp sdlhmi $PROGRAM_NAME/usr/bin/
+mv * $PROGRAM_NAME/usr/share/sdlhmi/; cp -r $PROGRAM_NAME/usr/share/sdlhmi/* .
```

Comment:
@LuxoftAKutsan what a trick is performed here? Are you trying to move the same files from root folder to sdlhmi folder and then copy them back?

Reply:
Fixed